### PR TITLE
remove Project language defaulting to TypeScript.  

### PR DESCRIPTION
**Pull request by cjus**
This involved adding a custom Deserializer.

### CODE REVIEW

File: apps/framework-cli/src/project.rs [hunk 1]
```diff
@@ -110,6 +110,53 @@ pub struct Project {
...
+
+        let language_project_config = match helper.language {
+            SupportedLanguages::Typescript => {
+                // Here, you would load or initialize the TypeScript project configuration
+                LanguageProjectConfig::Typescript(TypescriptProject::new(
+                    "default_name".to_string(),
+                ))
+            }
+            SupportedLanguages::Python => {
+                // Here, you would load or initialize the Python project configuration
```
Comment (phiSgr) [inline apps/framework-cli/src/project.rs:141]: This looks like WIP comments. Do you mean to fix it up here or are the values populated elsewhere?

Comment (cjus) [reply apps/framework-cli/src/project.rs:141]: No I should remove the comments.

**Comment by callicles** [comment]
I believe we can close this without merging. Here is the reasoning:

In the `load` method, on the `Project` object that loads a project from disk
* we first load the project using a default value for the config (Typescript). 
* Then we load the language specific config based on the language declared in the `moose.project.toml`
* Then we mutate what was previously loaded with the Typescript config to be the proper config

Ref: https://github.com/514-labs/moose/blob/e5cf777e4abaae5ab2b78894c28de4814c172c51/apps/framework-cli/src/project.rs#L177-L212

As such I don't think we need a custom deserializer.

**Comment by cjus** [comment]
I see. Closing this PR.

**Pull request closed without merging**